src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -154,68 +154,76 @@ export class MoveModal extends React.Component {
-          key={`content-${folderId}`}
-        >
-          {({ data, fetchStatus, hasMore, fetchMore }) => {
-            return (
-              <Explorer>
-                <Loader fetchStatus={fetchStatus} hasNoData={data.length === 0}>
-                  <FileList
-                    files={data}
-                    targets={entries}
-                    navigateTo={this.navigateTo}
-                  />
-                  <LoadMore hasMore={hasMore} fetchMore={fetchMore} />
-                </Loader>
-              </Explorer>
-            )
-          }}
-        </Query>
-        <RefreshableSharings>
-          {({ refresh }) => (
-            <Footer
-              onConfirm={() => this.moveEntries(refresh)}
-              onClose={onClose}
-              targets={entries}
-              currentDirId={folderId}
-              isMoving={isMoveInProgress}
-            />
-          )}
-        </RefreshableSharings>
-      </Modal>
+      <FixedDialog
+        open
+        onClose={onClose}
+        size="large"
+        classes={{ paper: 'u-h-100' }}
+        title={
+          <>
+            <Header entries={entries} />
+            <Query
+              query={folderQuery.definition()}
+              fetchPolicy={folderQuery.options.fetchPolicy}
+              as={folderQuery.options.as}
+              key={`breadcrumb-${folderId}`}
+            >
+              {({ data, fetchStatus }) => (
+                <Topbar
+                  navigateTo={this.navigateTo}
+                  currentDir={data}
+                  fetchStatus={fetchStatus}
+                />
+              )}
+            </Query>
+          </>
+        }
+        content={
+          <Query
+            query={contentQuery.definition()}
+            fetchPolicy={contentQuery.options.fetchPolicy}
+            as={contentQuery.options.as}
+            key={`content-${folderId}`}
+          >
+            {({ data, fetchStatus, hasMore, fetchMore }) => {
+              return (
+                <Explorer>
+                  <Loader
+                    fetchStatus={fetchStatus}
+                    hasNoData={data.length === 0}
+                  >
+                    <FileList
+                      files={data}
+                      targets={entries}
+                      navigateTo={this.navigateTo}
+                    />
+                    <LoadMore hasMore={hasMore} fetchMore={fetchMore} />
+                  </Loader>
+                </Explorer>
+              )
+            }}
+          </Query>
+        }
+        actions={
+          <RefreshableSharings>
+            {({ refresh }) => (
+              <Footer
+                onConfirm={() => this.moveEntries(refresh)}
+                onClose={onClose}
+                targets={entries}
+                currentDirId={folderId}
+                isMoving={isMoveInProgress}
+              />
+            )}
+          </RefreshableSharings>
+        }
+      />
     )
   }
 }
```

Comment:
is it validated by @joel-costa ?

Reply:
Nous n'avons pas de specs pour la MoveModal, @joel-costa est ok pour garder l'ancien style pour l'instant (ce qui est le cas ici)